src/main/java/com/solvd/laba/pages/HomePage.java

```diff
@@ -5,15 +5,23 @@
 import org.openqa.selenium.support.FindBy;
 import org.openqa.selenium.support.PageFactory;
 
+import com.solvd.laba.components.HeaderComponent;
+
 public class HomePage extends AbstractPage {
 
     @FindBy(xpath = "//h2[contains(text(), 'Full-Fledged practice website')]")
     private WebElement homePageHeading;
 
+    private HeaderComponent headerComponent;
 
     public HomePage(WebDriver driver) {
         super(driver);
         PageFactory.initElements(driver, this);
+        headerComponent = new HeaderComponent(driver);
+    }
+
+    public HeaderComponent getHeaderMenuComponent() {
```

Comment:
If the header is always present, you could move HeaderComponent to a base page class.